src/components/calcite-loader/calcite-loader.scss

```diff
@@ -1,3 +1,13 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --loader-font-size: when type is determinate, the font-size of the loader percentage
```

Comment:
Should all css vars be prefixed with `--calcite` because these are global and could conflict with other libraries?

Reply:
That makes sense to me. Can I just focus on the ones documented in this pr for now? (And create a new issue for the remainders)

Reply:
Actually it looks like the only stragglers are in split-button.scss, so I'll go ahead and rename them. (I used `\s--[^calcite].*:` to grep through the stylesheets so hopefully didn't miss any.)